jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -228,6 +234,22 @@ private void setMaxLineResults() {
     }
   }
 
+  /**
+   * Fetch ROWS_FETCH_SIZE_KEY value from property file and set it to
+   * "rowsFetchSize" value.
+   */
+  private void setRowsFetchSize() {
+    rowsFetchSize = Integer.valueOf(getProperty(ROWS_FETCH_SIZE_KEY));
```

Comment:
I think to preserve backward compatibility, it is better to first check for the presence of `ROWS_FETCH_SIZE_KEY / MAX_ROWS_KEY` in the properties. If there is no parameter, specify a reasonable default.
Examples:

- [JDBCInterpreter.java:L259 defaultIfEmpty](https://github.com/apache/zeppelin/blob/58068816a341c96aa96a08663a16eb89a7d85904/jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java#L259)
- [JDBCInterpreter.java:L221 if (!isEmpty(getProperty](https://github.com/apache/zeppelin/blob/58068816a341c96aa96a08663a16eb89a7d85904/jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java#L221)

Reply:
Regarding the default, you can use `getProperty(key, defaultValue)`

Reply:
Problem addressed. See the latest commit.